.github/pull_request_template.md

```diff
@@ -1,14 +1,25 @@
-## Overview: ##
+## Overview
 
-## Related Jira tickets: ##
+
+
+## Related
 
 * [FP-123](https://jira.tacc.utexas.edu/browse/FP-123)
+<!--
+- requires|required by|mimics https://github.com/TACC/.../pull/...
+-->
+
+## Changes
+
 
-## Summary of Changes: ##
```

Comment:
I like dropping this. 👍 .  so we would be say anything we want in the overview  and can write any "big picture" summary info

Reply:
I, personally, also **like** dropping "… Changes". Because I prefer to __either__ add my own review to my own PR with code-specific comments (I often do this) __or__ add my changes in a (hidden by default using `<details>`) list under Overview (I rarely do this, because I have "Changes" section).

But other developers may be disinclined to list specific changes in any of those three ways. I rarely read them, but I maybe other reviewers benefit.

Reply:
To simplify, questions we could ask:

1. Do we generally read or ignore "… Changes"?
2. If useful, should they be suggested in another way? (self-review notes, list in overview as necessary)

Reply:
good questions.

> 1 Do we generally read or ignore "… Changes"?

I seem to ignore them but sometimes go and reread them if I'm not following things.

> 2 If useful, should they be suggested in another way? (self-review notes, list in "Overview" as necessary)
I like the 2 but can't think of what would be in the self-review notes.  maybe just summarize the changes and what's trying to be solved. 🤷

Reply:
but your current changes look good so just gonna approve now and can review if things change significantly.

Reply:
I would consider alternatives to be an either/neither option.
- For big PRs, I'd do self-review comments.
- For little PRs, I might add list in overview.
- For tiny ones, neither.